src/main/java/soot/AbstractASMBackend.java

```diff
@@ -215,7 +215,8 @@ private int getMinJavaVersion(SootClass sc) {
       }
     }
 
-    for (SootMethod sm : sc.getMethods()) {
+    List<SootMethod> clonedList = new ArrayList<>(sc.getMethods());
```

Comment:
Bitte JavaDoc: Wir brauchen das, weil wir beim Herausschreiben eventuell Methoden splitten müssen, die länger sind, als die JVM-Spec erlaubt. Das ist zwar noch Work in Progress, aber ohne Kommentar versteht das hier keiner.